numpy/add_newdocs.py

```diff
@@ -2949,6 +2951,11 @@ def luf(lamdaexpr, *args, **kwargs):
       File "<stdin>", line 1, in <module>
     ValueError: total size of new array must be unchanged
```

Comment:
Can we add an example here for reshaping a non-contiguous array? So:

```
>>> something
AttributeError: incompatible shape for a non-contiguous array
```

Reply:
With or without error?

Reply:
With error. We already have an example of a without-error case. Should have said _"non-contiguous array with an incompatible shape"_.

Reply:
All non-contiguos examples are a bit contrived. Here's another
```
>>> np.zeros((4,2))[::2].shape = (-1,)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: incompatible shape for a non-contiguous array
```
Changing the error message is a bit out of scope, but the current one seems OK to me, it is the `(-1,)` that is the incompatible shape.